src/tests/core/conftest.py

```
@@ -8,6 +8,17 @@
 import pytest
 
 
+@pytest.fixture
+def logged_in_client(client, django_user_model):
+    """Return a client with an authenticated user."""
+    user = django_user_model.objects.create_user(
+        username="testuser",
+        password="testpass123",
```

Ruff (0.14.10)

Possible hardcoded password assigned to argument: "password"

(S106)